integrationTests/vm/testInitializer.go

```diff
@@ -526,16 +506,16 @@ func CreateTxProcessorWithOneSCExecutorMockVM(
 		ArgsParser:                     smartContract.NewArgumentParser(),
 		ScrForwarder:                   &mock.IntermediateTransactionHandlerMock{},
 		EpochNotifier:                  forking.NewGenericEpochNotifier(),
-		PenalizedTooMuchGasEnableEpoch: argEnableEpoch.PenalizedTooMuchGasEnableEpoch,
-		MetaProtectionEnableEpoch:      argEnableEpoch.MetaProtectionEnableEpoch,
-		RelayedTxEnableEpoch:           argEnableEpoch.RelayedTxEnableEpoch,
+		PenalizedTooMuchGasEnableEpoch: enableEpochs.PenalizedTooMuchGasEnableEpoch,
+		MetaProtectionEnableEpoch:      enableEpochs.MetaProtectionEnableEpoch,
+		RelayedTxEnableEpoch:           enableEpochs.RelayedTransactionsEnableEpoch,
 	}
 
 	return transaction.NewTxProcessor(argsNewTxProcessor)
 }
 
 // CreateOneSCExecutorMockVM -
-func CreateOneSCExecutorMockVM(accnts state.AccountsAdapter) vmcommon.VMExecutionHandler {
+func CreateOneSCExecutorMockVM(accnts state.AccountsAdapter, enableEpochs config.EnableEpochs) vmcommon.VMExecutionHandler {
```

Comment:
`enableEpochsConfig`, but can also stay as it is.

Reply:
Maybe @dragos-rebegea will rename when it will need to refactor this area